common/src/main/java/com/wynntils/core/config/objects/ConfigOptionHolder.java

```diff
@@ -56,6 +57,7 @@ public Object getValue() {
     public void setValue(Object value) {
         try {
             FieldUtils.writeField(optionField, parent, value, true);
+            parent.onUpdate();
```

Comment:
Can we pass the optionField as a argument to onUpdate(), please? The callee can then chose to ignore it, or act on it.